src/app/ApplicationTemplate.UWP/Views/Styles/Controls/MenuRadioButton.xaml

```diff
@@ -6,11 +6,6 @@
 
 	<Style x:Key="DefaultMenuRadioButtonStyle"
 		   TargetType="c:MenuRadioButton">
-
-		<Setter Property="Foreground"
```

Comment:
What is the default value for foreground here? Why not use one of Material brush?

Reply:
Oh I see, value is set via the Style.Setters below